src/client/Microsoft.Identity.Client/PlatformsCommon/Interfaces/IPlatformProxy.cs

```diff
@@ -112,7 +112,7 @@ internal interface IPlatformProxy
         /// does not support silent auth and gives the RT back to MSAL.
         /// </summary>
         /// <returns></returns>
-        bool CanBrokerSupportSilentAuth();
+        bool CanBrokerSupportSilentAuth(); // TODO: get rid of this!
```

Comment:
>// [](start = 43, length = 2)

lol. is there an issue tracking that work?

Reply:
I'll remove the comment for now. "No refactoring on the backlog" is what I heard from a knowledgeable PM.